maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -560,6 +562,24 @@ class Orchestra(
         return mutatiing
     }
 
+    private fun retryCommand(command: RetryCommand, config: MaestroConfig?): Boolean {
+        val maxAttempts = command.maxAttempts?.toIntOrNull() ?: 2
+
+        var attempt = 0
+        while(attempt < maxAttempts) {
+            try {
+                return runSubFlow(command.commands, config, command.config)
+            } catch (exception: Throwable) {
+                val message = "Retrying the commands due to an error while execution"
```

Comment:
Should we show the error in the insight too?

Reply:
Done ✅